docs/src/architecture/architecture.md

```diff
@@ -1,2 +1,122 @@
-# ADRIA Architecture
+# Architectural overview
 
+ADRIA consists of three overarching components:
+
+1. a set of Multi-Criteria Decision Analysis methods used to emulate decision-makers
```

Comment:
Extremely picky here, but maybe "simulate" instead of "emulate", emulate suggests a perfect model to me.